pkg/cmd/rollbackcopy/fileioutils_test.go

```diff
@@ -0,0 +1,163 @@
+package rollbackcopy
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_checkAndCreateDir(t *testing.T) {
+	type args struct {
+		dirName string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := checkAndCreateDir(tt.args.dirName); (err != nil) != tt.wantErr {
+				t.Errorf("checkAndCreateDir() error = %v, wantErr %v", err, tt.wantErr)
+			}
+		})
+	}
+}
+
+func Test_dirExists(t *testing.T) {
+	type args struct {
+		dirname string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got, _ := dirExists(tt.args.dirname); got != tt.want {
+				t.Errorf("dirExists() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_findTheLatestRevision(t *testing.T) {
+	type args struct {
+		dir     string
+		podname string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
again are we doing these? I see no reason to ship these tests if they are not going to be populated.

Reply:
Pushed a new commit deleting the file.